wandb/sdk/artifacts/_utils.py

```diff
@@ -0,0 +1,58 @@
+"""Internal helper functions for artifacts."""
+
+from __future__ import annotations
+
+import re
+from typing import TYPE_CHECKING, Hashable, TypeVar
+
+if TYPE_CHECKING:
+    from typing import Collection, Iterable, Iterator
+
+
+HashableT = TypeVar("HashableT", bound=Hashable)
+
+
+def validate_aliases(aliases: Collection[str]) -> list[str]:
+    """Validate the given artifact aliases, returning them as a list if successful.
```

Comment:
```suggestion
    """Validates the artifact aliases and returns them as a list.
```

"if successful" is redundant---it won't return at all if it's not successful

Reply:
Good point -- slightly edited for [D401](https://docs.astral.sh/ruff/rules/non-imperative-mood/) by habit, but much clearer/shorter!

Reply:
nice, didn't know that imperative mood was recommended!